mitosheet/src/mito/utils/actions.tsx

```diff
@@ -321,6 +322,28 @@ export const getActions = (
                 windows: 'Ctrl+C'
             }
         },
+        [ActionEnum.CopyCode]: {
+            type: 'build-time',
+            staticType: ActionEnum.CopyCode,
+            icon: CopyIcon,
+            toolbarTitle: 'Copy Code to Clipboard',
+            longTitle: 'Copy Code to Clipboard',
+            actionFunction: () => {
+                closeOpenEditingPopups();
+
+                const stringToCopy = analysisData.code.join('\n');
```

Comment:
There used to be a util for building a code string. Does this have the imports and stuff? I think probably, as the refactor moved that to the backend.